test/integration/preflight/check-yum-update/test-yum-repo-broken.yml

```diff
@@ -0,0 +1,16 @@
+- hosts: localhost
+  tasks:
+
+  - include: tasks/create-test-container.yml
+
+  - name: break the local yum repo
+    delegate_to: dockertest
+    replace: dest=/etc/yum.repos.d/local.repo backup=no
+             regexp='^baseurl'
+             replace='#baseurl'
```

Comment:
Hmm, I wonder when we will decide to have separate images vs. patch the image like this.